Source/Kesmai.Server/Game/Items/Equipment/Weapons/Weapon.cs

```diff
@@ -59,9 +59,22 @@ public override int CalculateBlockingBonus(ItemEntity item)
 		/// </summary>
 		public override double CalculateFumbleChance(MobileEntity entity)
```

Comment:
I've been away, but og, and lands don't make 2h necessarily mean 100% fumble like this.  OG, all 2h weapons became 1h weapons eventually, they just got nerfed slightly because of it.  Lands, some 2h weapons were viable 1h and some were true 2h weapons, and it removed the skill portion.  Are we intending to make all 2h true 2h?

Reply:
Yes. We can do fumble or absolutely no skill gain from it.

Reply:
2 h has base defense by itself